Add Spanish automatic translation (#381)
ropensci/roweb3: content/blog/2022-04-19-software-review-editorial-challenges-es/index.md

Para aumentar la dificultad de encontrar revisores disponibles, tenemos una lista de [criterios para elegir a los revisores](https://devguide.ropensci.org/editorguide.html#criteria-for-choosing-a-reviewer) para garantizar la diversidad (tanto en habilidades como en antecedentes), y para crear continuamente un grupo de revisores experimentados sin sobrecargar a ninguno de ellos.

El año pasado lanzamos un [nuevo formulario para el voluntariado de revisores](/blog/2021/11/18/devguide-0.7.0/#a-new-form-for-volunteer-reviewing) que nos permite recoger datos más precisos sobre la experiencia técnica y temática, así como una pregunta opcional: "¿Te consideras parte de un grupo subrepresentado en los campos de la ciencia de los datos, la programación o en tu campo de trabajo principal?
Mantenemos esta información en una base de datos (gestionada con Airtable), junto con el historial de revisiones de cada revisor.  Esta base de datos es un recurso para los editores cuando buscan revisores con una experiencia, una formación o una disponibilidad particulares.

> Cuando esto no es suficiente, por ejemplo, después de repetidas negativas o falta de respuesta de posibles revisores, los editores pueden pedir recomendaciones a otros editores en nuestro canal de Slack. (Algunos de nosotros somos editores de revistas científicas tradicionales, y desearíamos tener una charla tan útil con nuestros coeditores allí).

Reviewer: ```suggestion
Cuando esto no es suficiente, por ejemplo, después de repetidas negativas o falta de respuesta de posibles revisores/as, los/as editores/as pueden pedir recomendaciones al resto del equipo editorial en nuestro canal de Slack. (Algunos de somos editores/as de revistas científicas tradicionales, y desearíamos tener una charla tan útil con nuestros coeditores/as allí).
```

El objetivo es no retrasar demasiado el proceso de revisión.

## Autores que abandonan

Es una situación más triste cuando el paquete *autores* dejan de responder, ya que hay que detener el proceso de revisión.
Es especialmente decepcionante cuando ocurre después de las revisiones, ya que los revisores pueden sentir que su trabajo se ha desperdiciado.
Al final, no podemos hacer nada al respecto, ya que está fuera del control de nuestros editores y revisores (y de los autores).

Sin embargo, para evitar problemas sencillos con comunicaciones perdidas, recomendamos que los autores se aseguren de recibir las notificaciones de GitHub, ya que intentamos que la comunicación sea transparente a través de GitHub y no solemos utilizar el correo electrónico.
Para hacer frente a problemas más importantes relacionados con el tiempo y el compromiso, hemos añadido recientemente frases a (la versión de desarrollo de) la [guía del autor](https://devdevguide.netlify.app/authors-guide.html) y a las plantillas de envío: "Espero mantener este paquete durante al menos 2 años o encontrar un sustituto".

> Esto debería incitar a los autores de paquetes a pensar en el mantenimiento del paquete a largo plazo.

Reviewer: Esta frase es casi idéntica a la posterior, está en el post de la versión en inglés, pero es repetitiva.  Yo dejaría la posterior y borraría esta.

Aunque esto no evitará todos los problemas, esperamos que incite a los autores de paquetes a pensar en el mantenimiento del paquete a largo plazo antes de enviarlo para su revisión.

También entendemos que, aunque los autores tengan buenas intenciones, la vida puede torcerse inesperadamente.

> Por lo tanto, otro esfuerzo que hacemos es poner retenciones en las revisiones (aplicando etiquetas de retención) según sea necesario.

Reviewer: ```suggestion
Por lo tanto, otro esfuerzo que hacemos es poner las revisiones "en espera" (aplicando etiquetas de "en espera") según sea necesario.
```

> Estas retenciones se revisan cada tres meses, hasta un año.

Reviewer: ```suggestion
Estas etiquetas de trabajo suspendido se revisan cada tres meses y hasta un año.
```

> ## Sobrecarga de trabajo de los editores (y revisores)

Reviewer: ```suggestion
## Sobrecarga de trabajo de editores/as (y revisores/as)
```

La revisión de software requiere el trabajo de todos: editores, revisores y, obviamente, autores.

> Para reducir el trabajo de los editores en particular, hemos mejorado la automatización, que fue el tema de una convocatoria de la comunidad, ["Mejorar la revisión por pares del software con la automatización de GitHub"](/commcalls/dec2021-automation/).

Reviewer: ```suggestion
Para reducir el trabajo de quienes editan en particular, hemos mejorado la automatización, que fue el tema de una charla con la comunidad, ["Mejorar la revisión por pares del software con la automatización de GitHub"](/commcalls/dec2021-automation/).
```

> rOpenSci ha trabajado con The Journal of Open Source Software para ampliar el enfoque de JOSS de la publicación impulsada por chatops en un nuevo chat-bot de GitHub que gestiona nuestro proceso editorial: asignar tareas, etiquetar temas, ejecutar pruebas en los envíos de software, devolver informes a los revisores y editores, y registrar las revisiones en una base de datos externa (Airtable). Todo ello desde la comodidad de un comentario de una cuestión de GitHub.

Reviewer: ```suggestion
rOpenSci ha trabajado con The Journal of Open Source Software para ampliar el enfoque de JOSS de la publicación impulsada por chatops en un nuevo chat-bot de GitHub que gestiona nuestro proceso editorial: asignar tareas, etiquetar temas, ejecutar pruebas en los envíos de software, devolver informes a quienes revisan y editan, y registrar las revisiones en una base de datos externa (Airtable). Todo se puede hacer desde la comodidad de un comentario en un issue de GitHub.
```

> Por ejemplo, podrías clonar un repositorio localmente, instalar dependencias, ejecutar comprobaciones y publicar manualmente los resultados... **o** podrías simplemente escribir lo siguiente en un comentario de incidencia de GitHub:

Reviewer: ```suggestion
Por ejemplo, podrías clonar un repositorio localmente, instalar dependencias, ejecutar comprobaciones y publicar manualmente los resultados... **o** podrías simplemente escribir lo siguiente en un comentario en un _issue_ de GitHub:
```

```

> @ropensci-review-bot comprobar paquete

Reviewer: ```suggestion
@ropensci-review-bot check package
```

```

> Del mismo modo, puedes utilizar lo siguiente para registrar un revisor en los metadatos de la cuestión enviada, así como en nuestra base de datos Airtable.

Reviewer: ```suggestion
Del mismo modo, puedes utilizar el siguiente comando para registrar a la persona que revisa en los metadatos del _issue_ de la revisión, así como en nuestra base de datos Airtable.
```

```

> @ropensci-review-bot añade a @maelle a los revisores

Reviewer: ```suggestion
@ropensci-review-bot add @maelle to reviewers
```

```

> ¡Una gran parte de la limitación de la carga de trabajo de los editores es tener suficientes editores!  Hemos estado [ampliando nuestro equipo editorial](/tags/editors/)con el objetivo de que los editores sólo se ocupen de [unos 4 envíos al año](https://devdevguide.netlify.app/editorguide.html#editors-responsabilities).  

Reviewer: ```suggestion
Una gran parte de la limitación de la carga de trabajo de quienes editan es ¡tener suficientes personas en ese rol!  Hemos estado [ampliando nuestro equipo editorial](/tags/editors/) con el objetivo de que los/as editores/as sólo se ocupen de [unos 4 envíos al año](https://devdevguide.netlify.app/editorguide.html#editors-responsabilities).  
```

> A medida que nuestro volumen de envíos crece y los editores van rotando (pedimos compromisos de 2 años con opción de renovación), reclutamos nuevos editores en gran parte de nuestro grupo de revisores.

Reviewer: ```suggestion
A medida que nuestro volumen de envíos crece y las personas que editan van rotando (pedimos compromisos de 2 años con opción de renovación), reclutamos nuevos/as editores/as en gran parte de nuestro equipo de revisión.
```

> Invitamos a los revisores con experiencia a editar como invitados y les preguntamos si quieren quedarse si todo funciona.  

Reviewer: ```suggestion
Invitamos a los/as revisores/as con experiencia a editar como invitados/as y les preguntamos si quieren quedarse si todo funciona bien.  
```

> Nuestra [guía del editor](https://devguide.ropensci.org/editorguide.html) facilita la incorporación... y la incorporación de nuevos editores da lugar a comentarios sobre esa guía y, por tanto, a su mejora.

Reviewer: ```suggestion
Nuestra [guía para editores/as](https://devguide.ropensci.org/editorguide.html) facilita la incorporación de las nuevas personas al equipo... y la incorporación de nuevas personas que van a editar da lugar a comentarios sobre esa guía y, por tanto, a su mejora.
```

> Si quieres probar a editar [empieza por ofrecerte como revisor](/software-reviewer)¡!

Reviewer: ```suggestion
Si quieres probar editar ¡[empieza por ofrecerte para revisar](/software-reviewer)!
```

## Conclusión

> Aquí hemos resumido algunos problemas habituales de nuestros consejos de redacción.

Reviewer: ```suggestion
Hemos resumido algunos de los problemas habituales de nuestro equipo editorial.
```

> Hemos comprobado que es muy importante automatizar los procesos, pero también que los editores puedan comunicarse directamente entre sí para encontrar soluciones, así como para compadecerse y animarse mutuamente.

Reviewer: ```suggestion
Hemos comprobado que es muy importante automatizar los procesos, pero también que las personas que editan puedan comunicarse directamente entre sí para encontrar soluciones, así como para acompañarse y animarse mutuamente.
```

A medida que continúe la revisión del software, seguramente nos encontraremos con nuevos retos que abordar.

> Esperamos que nuestras experiencias sean útiles para otros editores y que puedan ayudar a informar a otros consejos editoriales.

Reviewer: ```suggestion
Esperamos que nuestras experiencias sean útiles para otros/as editores/as y que puedan ayudar a informar a otros consejos editoriales.
```
